extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduOperations.java

```diff
@@ -109,33 +107,7 @@ public boolean deleteAll(
       final String typeName,
       final Short adapterId,
       final String... additionalAuthorizations) {
-    KuduSession session = getSession();
```

Comment:
Getting rid of this, because the method isn't fully implemented (doesn't use `typeName`), so we don't want anyone accidentally using it. The Cassandra datastore also does not implement this and returns false.

Reply:
Can you instead comment this out and add a `TODO`?